src/elements/toolbar.js

```diff
@@ -199,7 +236,15 @@ export default class LexicalToolbarElement extends HTMLElement {
 
   #resetToolbar() {
     while (this.#overflowMenu.children.length > 0) {
-      this.insertBefore(this.#overflowMenu.children[0], this.#overflow);
+      var child = this.#overflowMenu.children[0]
+
+      // Move undo and redo after spacer, and the rest before spacer
+      var to = this.#spacer
```

Comment:
Could you move the spacer to the menu to represent it there too? I would love if we could remove the spacer-specific rearrangement logic, and we just moved elements from one place to another, in a generic way. We should let you use multiple spacers in the menu and fully configure the Lexxy menu at you want, so I would rather avoid expecting a special anchoring spacer element if possible. Also, I think these spacers can make sense in the collapsed menu too, right?